llama-index-integrations/llms/llama-index-llms-deepinfra/llama_index/llms/deepinfra/constants.py

```diff
@@ -8,3 +8,5 @@
 ENV_VARIABLE = "DEEPINFRA_API_TOKEN"
 """Default model name for DeepInfra embeddings."""
 DEFAULT_MODEL_NAME = "mistralai/Mixtral-8x22B-Instruct-v0.1"
+"""Default max tokens. DeepInfra documentation constant is set."""
+DEFAULT_MAX_TOKENS = 512
```

Comment:
Is this a good default value? I think OpenAI default is 4096

Reply:
eh, this is not a bad choice. It kind of depends on how big the overall context window is too. 

For openai, the default is -1 I think 🤔

Reply:
Anthropic also has 512 as default

Reply:
Lol oh -1 != 4096. I must've read an incorrect resource